asserts/asserts.go

```diff
@@ -333,6 +330,10 @@ func buildAndSign(assertType AssertionType, headers map[string]string, body []by
 	if err != nil {
 		return nil, fmt.Errorf("failed to sign assertion: %v", err)
 	}
+	// add an extra newline
+	temp := bytes.NewBuffer(signature)
+	temp.WriteByte('\n')
+	signature = temp.Bytes()
```

Comment:
Btw, the comment says exactly what the code is saying. The more interesting question to answer in the comment is why the extra line is being added.